api/config/service/auditsink/slack/v1/slack.proto

```diff
@@ -13,4 +13,36 @@ message SlackConfig {
   string channel = 2 [ (validate.rules).string = {min_bytes : 1} ];
 
   clutch.config.service.audit.v1.Filter filter = 3;
+
+  repeated CustomMessage overrides = 4;
+}
+
+// TODO: (sperry) update docs & add to clutch.sh
+/*
+By default, the sink provides a formatted message using a subset of info in an audit event. A custom
+message can alo be created for a /SERVICE/METHOD using the audit event's metadata (API Request/Response).
+The custom message will be appended to the default message for a richer slack audit.
+*/
+message CustomMessage {
```

Comment:
added more details and examples. in a follow up PR will work on some doc for clutch.sh